test/test_using_plams/test_opt_orca.py

```diff
@@ -45,10 +47,8 @@ def test_opt_orca():
 
 @pytest.mark.slow
 def test_methanol_opt_orca():
-    """
-    Run a methanol optimization and retrieve the optimized geom.
-    """
-    methanol = Molecule('test/test_files/methanol.xyz')
+    """Run a methanol optimization and retrieve the optimized geom."""
+    methanol = Molecule((PATH_MOLECULES / "methanol.xyz"))
```

Comment:
The double parenthesis are redundant here.

Reply:
Thanks, I forgot them!